README.md

```diff
@@ -8,19 +8,15 @@ It will be used by the Firefox Accounts pairing flow, with one side
 of the channel being web content from https://accounts.firefox.com and
```

Comment:
let's add a doc about running `npm ci` and `npm run build` to ensure the build is consistent and no new sub-deps show up. Maybe we can do it as a separate PR.

Reply:
Maybe this should be a pre-commit hook or something?  (TBH it kind of weirds me out checking in the build output on every commit, and I wonder if we can only do it on releases or something; but if we're going to do it we should be able to automate it)

Reply:
pre-commit hooks are local only, you want something in CI ideally